test/f90_correct/len_intrinsic.f90

```diff
@@ -0,0 +1,47 @@
+!** Copyright (c) 2019, Arm Ltd.  All rights reserved.
```

Comment:
This file should be moved to test/llvm_ir_correct/. Use the same license notice as in other files.

Reply:
done.